packages/runner-ct/src/app/RunnerCt.scss

```diff
@@ -12,6 +12,10 @@ main.app-ct {
   font-size: $font-size;
 }
 
+.d-none {
```

Comment:
IMHO this name can be unclear when reading it from the code, maybe we can create a more descriptive name? For reference: tailwind css using `hidden` for this [source](https://tailwindcss.com/docs/display)

Reply:
How about `display-none`? 🤔

Reply:
Changed to display-none... I think this is much more clear than hidden - it's literally the exact css, `display: none`. 

We can revisit these at some point, we should probably grab a library that has these to save us the effort of discussing class names (eg tailwind, but probably something a little smaller).